src/components/RJST/components/Table/TableActions.tsx

```diff
@@ -33,6 +29,45 @@ interface TableActionsProps<T> {
 	onColumnPreferencesChange?: ColumnPreferencesChangeProp<T>;
 }
 
+const SortableItem = ({ column, handleColumnSelection }: any) => {
+	const { state: analyticsState } = useAnalyticsContext();
+	const { attributes, listeners, setNodeRef, transform, transition } =
+		useSortable({ id: column.key });
+
+	return (
+		<MenuItem
+			ref={setNodeRef}
+			{...attributes}
+			sx={{ transform: CSS.Transform.toString(transform), transition }}
+		>
+			{analyticsState.featureFlags?.columnOrdering && (
+				<ListItemIcon
+					{...listeners}
+					sx={{
+						cursor: 'grab',
+						pr: 2,
+						mx: 0,
+						minWidth: 'auto !important',
+					}}
+				>
+					<FontAwesomeIcon icon={faGripVertical} />
+				</ListItemIcon>
+			)}
+			<FormControlLabel
+				sx={{ flex: 1, width: '100%', m: 0 }}
+				control={
+					<Checkbox
+						sx={{ m: 0 }}
+						onClick={() => handleColumnSelection(column)}
+						checked={column.selected}
+					/>
+				}
+				label={typeof column.label === 'string' ? column.label : column.title}
```

Comment:
```suggestion
				label={column.label ?? column.title}
```

Reply:
this is not the same thing, we are checking if the actions is a JSX element or a string as in this dropdown we only want to show the string and not the entire JSX element that might contain a tootlip or other stuff